### PR TITLE
feat: add skip_iam_grants flag

### PR DESCRIPTION
**Pull request by mikelaramie**
## Summary
This PR adds a `skip_iam_grants` variable to the Config integration to allow teams who manage IAM-policy-as-code external to this provider to skip the IAM grant sections of the integration

## How did you test this change?

Tested the integration with my demo org, adding the IAM grants outside of the terraform code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,10 +64,10 @@ cloudasset.googleapis.com
 
 | Name | Version |
 |------|---------|
-| <a name="provider_google"></a> [google](#provider\_google) | >= 4.4.0, < 5.0.0 |
-| <a name="provider_lacework"></a> [lacework](#provider\_lacework) | ~> 0.2 |
-| <a name="provider_random"></a> [random](#provider\_random) | n/a |
-| <a name="provider_time"></a> [time](#provider\_time) | ~> 0.6 |
+| <a name="provider_google"></a> [google](#provider\_google) | 4.27.0 |
```
Comment (dmurray-lacework) [inline README.md]: These look just to be local versions. Will need to revert these

File: README.md [hunk 2]
```diff
@@ -110,6 +110,7 @@ cloudasset.googleapis.com
...
 | <a name="input_service_account_name"></a> [service\_account\_name](#input\_service\_account\_name) | The Service Account name (required when use\_existing\_service\_account is set to true). This can also be used to specify the new service account name when use\_existing\_service\_account is set to false | `string` | `""` | no |
 | <a name="input_service_account_private_key"></a> [service\_account\_private\_key](#input\_service\_account\_private\_key) | The private key in JSON format, base64 encoded (required when use\_existing\_service\_account is set to true) | `string` | `""` | no |
+| <a name="input_skip_iam_grants"></a> [skip\_iam\_grants](#input\_skip\_iam\_grants) | Skip generation of custom role, and IAM grants to the Service Account, for customers who use IAM policy-as-code external to the Lacework module.  WARNING - integration will fail if grants are not in place prior to execution. | `bool` | `false` | no |
```
Comment (rmoles) [inline README.md]: Wondering if we are expecting that this should only be used with an existing Service Account?
`WARNING - integration will fail if grants are not in place prior to execution.` - Prior to the Terraform Apply or prior to a compliance scan?

Comment (dmurray-lacework) [reply README.md]: So it must be used in conjunction with bring your own svc account

File: main.tf [hunk 3]
```diff
@@ -18,13 +18,13 @@ locals {
     base64decode(module.lacework_cfg_svc_account.private_key)
   ))
 
-  default_project_roles = [
+  default_project_roles = var.skip_iam_grants ? [] : [
```
Comment (rmoles) [inline main.tf]: We should add a local `skip_iam_grants` that checks whether an existing Service Account is being used also. It looks as though this method would only work when providing an existing SA.

**Review by dmurray-lacework — APPROVED** [review]

**Pull request merged**